Remove "simple" part 4: change to "basic" (#36763)
mdn/content: files/en-us/web/html/element/datalist/index.md

Predefined values can be shown there, allowing the user to quickly fill the control value.

> [!NOTE]

> > When type is not supported, `text` type creating basic text field will be used instead. That field will correctly recognize recommended values and display them to the user in a drop-down menu.

Reviewer: ```suggestion
> When these types are not supported, a basic `text` type will be rendered instead, creating a text field. That field will correctly recognize recommended values and display them to the user in a drop-down menu.
```
